src/index.css

```diff
@@ -276,12 +291,37 @@ h1 {
 
 .poke-name {
     text-transform: uppercase;
+    text-align: center;
 }
 
 .shiny-status {
     text-transform: none;
 }
 
+.current-hand img{
+  padding:0 7px;
+}
+
+/* Button styles */
+
+button {
+padding:10px;
+width:90px;
+border-radius: 10px;
+transition:0.7s;
+cursor:pointer;
+}
+
+button:hover{
+  background-color: #ffff00;
+}
+
+button:focus{
+  background-color: #ffff00;
```

Comment:
In my opinion, the buttons should only be yellow when hovered over. I don't think they should remain yellow once they're clicked, it makes for confusing UX.